apps/src/lib/ui/lti/sync/LtiSectionSyncDialog.tsx

```diff
@@ -198,13 +227,30 @@ export default function LtiSectionSyncDialog({
   const hideCloseButton = currentView === SubView.SPINNER;
 
   const SectionRow = (id: string, section: LtiSection) => {
-    const sectionOwnerName = section.instructors.find(
-      instructor => instructor.isOwner
-    )?.name;
+    const instructorOptions = section.instructors.map(instructor => ({
+      value: instructor.id.toString(),
+      text: instructor.name,
+    }));
     return (
       <tr key={id} role={'gridcell'}>
         <td style={styles.tableCellText}>{section.short_name}</td>
-        <td style={styles.tableCellText}>{sectionOwnerName}</td>
+        <td style={styles.tableCellText}>
+          <SimpleDropdown
+            name={`instructor-dropdown-${id}`}
+            items={instructorOptions}
+            selectedValue={sectionOwners[section.lti_section_id].toString()}
+            labelText={i18n.ltiSectionSyncDialogHeaderPrimaryInstructor()}
+            isLabelVisible={false}
+            size={'xs'}
+            onChange={e =>
+              setSectionOwners(owners => {
+                return Object.assign({}, owners, {
+                  [section.lti_section_id]: parseInt(e.target.value, 10),
+                });
```

Comment:
nit: can be shortened

```suggestion
              setSectionOwners(owners => ({
                ...owners,
                [section.lti_section_id]: parseInt(e.target.value, 10),
              }))
```